extensions/traceability.sigmf-ext.md

```diff
@@ -0,0 +1,199 @@
+# The `traceability` SigMF Extension Namespace v1.0.0
+
+This document proposes a new extension namespace called `traceability` for the Signal Metadata
+Format (SigMF) specification. This extension provides traceability information for the metadata.
+
+## 1 Global
+
+`traceability` extends the [Global](https://github.com/gnuradio/SigMF/blob/master/sigmf-spec.md#global-object) object.
+
+The following fields are added to the `global` object:
+
+|name|required|type|description|
+|----|--------|----|-----------|
+|`traceability:last_modified`|false|[DataChange](#datachange-object)|Captures the author and timestamp of the most recent modification|
+|`traceability:last_reviewed`|false|[DataChange](#datachange-object)|Captures the author and timestamp of the most recent review|
+|`traceability:revision`|false|integer|Specifies the revision number associated with the metadata|
+|`traceability:origin`|false|[Origin](#origin-object)|Provides information about the origin of the data|
+
+### DataChange Object
+
+|name|required|type|description|
+|----|--------|----|-----------|
+|`author`|false|string|Email address of the author who changed the metadata|
+|`time`|true|string (date-time)|Timestamp of the modification in ISO 8601 format|
```

Comment:
ok, i think this is actually my last comment @777arc thoughts on making this `datetime` to be consistent with the rest of the spec?

Reply:
Yeah consistency is important, plus its a datetime not just time